stdlib/test/collections/test_list.mojo

```diff
@@ -741,6 +741,19 @@ def test_list_mult():
     assert_equal(len(List[Int](1, 2, 3) * 0), 0)
 
 
+def test_list_contains():
+    var x = List[Int](1, 2, 3)
+    assert_false(0 in x)
+    assert_true(1 in x)
+    assert_false(4 in x)
```

Comment:
Normally we use `__contains__` directly in these test to avoid potential pitfalls from implicit conversions. In this case, I would recommend to do that, and maybe keep on instance of `0 in x` to make sure the syntax sugar works.

Reply:
Yes, sorry, i use the edit button on github because:
```
./scripts/run-tests.sh 
Creating build directory for building the stdlib running the tests in.
Mismatch in compiler versions! Cannot build the standard library.
Expected compiler version: 2024.5.1515
Current installed compiler version: 2024.5.1607
Please run modular update nightly/mojo to get the latest compiler.
```